assets/strings/eng.json

```diff
@@ -61,5 +61,27 @@
     "Downloads_Para": "Current versions of the <b>Apertium toolbox</b> as well as of <b>language-pair data</b> are available from <a href='https://sourceforge.net/projects/apertium/' target='_blank' rel='noopener'>the SourceForge page</a>. Installation instructions for Apertium on all major platforms are provided on the <a href='http://wiki.apertium.org/wiki/Installation' target='_blank' rel='noopener'>Wiki's Installation page</a>.",
     "Contact_Para": "<h4>IRC channel</h4> <p>The quickest way to contact us is by joining our <a href='http://wiki.apertium.org/wiki/IRC' target='_blank' rel='noopener'><abbr title='Internet Relay Chat'>IRC</abbr> channel, #apertium</a> at irc.freenode.net, where users and developers of Apertium meet. You don't need an IRC client; you can use <a href='http://webchat.freenode.net/?channels=apertium'>freenode webchat</a>. </p> <h4>Mailing list</h4> <p>Also, subscribe to the <a href='https://lists.sourceforge.net/lists/listinfo/apertium-stuff' target='_blank' rel='noopener'> apertium-stuff mailing list</a>, where you can post longer proposals or issues, as well as follow general Apertium discussions. </p> <h4>Contact</h4> <p>Feel free to contact us via the <a href='mailto:apertium-contact@lists.sourceforge.net' target='_blank' rel='noopener'>apertium-contact mailing list</a> if you find a mistake, there's a project you would like to see us work on, or you would like to help out.</p>",
     "Install_Apertium": "Install Apertium",
-    "Install_Apertium_Para": "Are you experiencing slow responses? Our servers might be overloaded. Learn how to <a href='http://wiki.apertium.org/wiki/Installation' target='_blank' rel='noopener'>install Apertium locally</a>."
+    "Install_Apertium_Para": "Are you experiencing slow responses? Our servers might be overloaded. Learn how to <a href='http://wiki.apertium.org/wiki/Installation' target='_blank' rel='noopener'>install Apertium locally</a>.",
+    "Dictionary_Translations": "Dictionary translations of {{word}}",
+    "POS_n": "Noun",
```

Comment:
@jonorthwash I think I mentioned my concern with this earlier. Don't different language modules use different abbreviations?

Reply:
@sushain97 So how would I fix this?

Reply:
Not really for you to worry about for this task :)